api/src/main/java/org/openmrs/api/impl/ObsServiceImpl.java

```diff
@@ -102,6 +99,9 @@ public Obs saveObs(Obs obs, String changeMessage) throws APIException {
 		//Should allow updating a voided Obs, it seems to be pointless to restrict it,
 		//otherwise operations like merge patients won't be possible when to moving voided obs
 		if (obs.getObsId() == null || obs.getVoided()) {
+			if (obs.getObsId() == null){
```

Comment:
This is so that only new obs with  null ids are checked and not those which have been voided